src/renderer/components/cluster-view/EthPipeRenderer.tsx

```diff
@@ -100,6 +100,7 @@ const EthPipeRenderer: FC<EthPipeRendererProps> = ({
                             totalOps,
                             linkType: link.type,
                             totalDataBytes: link.totalDataBytes,
+                            initialMaxBandwidth: link.maxBandwidth,
```

Comment:
code duplication: `const { saturation } = calculateLinkSaturationMetrics({...` and the method is called twice

Reply:
It is calculating two times:
1. General node saturation
2. Normalized saturation for all links on the node

We may not need one of them, but didn't want to remove any logic and break things.